src/ontology/mondo-edit.obo

```diff
@@ -240166,6 +240176,7 @@ name: autosomal recessive limb-girdle muscular dystrophy type 2K
 def: "Autosomal recessive limb-girdle muscular dystrophy type 2K (LGMD2K) is a form of limb-girdle muscular dystrophy characterized by the onset of slowly progressive proximal muscle weakness during childhood (with fatigue and difficulty running and climbing stairs) and developmental delay. Mild intellectual deficit and microcephaly, without any obvious structural brain abnormality, are found in all patients. Mild pseudohypertrophy and joint contractures of the ankles have also been reported." [Orphanet:86812]
 subset: ordo_disease {source="Orphanet:86812"}
 synonym: "autosomal recessive limb-girdle muscular dystrophy caused by mutation in POMT1" EXACT [MONDO:design_pattern]
+synonym: "LGMD-POMT1 related" EXACT CLINGEN_PREFERRED []
```

Comment:
could you please add a dbxref here:
https://clinicalgenome.org/affiliation/50061/